pydantic_evals/pydantic_evals/evaluators/context.py

```
@@ -9,12 +9,13 @@
 from dataclasses import dataclass, field
 from typing import Any, Generic
 
+from pydantic import TypeAdapter
 from typing_extensions import TypeVar
 
 from ..otel._errors import SpanTreeRecordingError
 from ..otel.span_tree import SpanTree
 
-__all__ = ('EvaluatorContext',)
+__all__ = ('EvaluatorContext', 'EVALUATOR_CONTEXT_ADAPTER')
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)